main.py

```
@@ -112,16 +208,275 @@ async def validate_and_sync_packs(bot, user_id):
+        try:
+            with sqlite3.connect(DB_FILE) as conn:
+                c = conn.cursor()
+                c.execute(
+                    "UPDATE sticker_drafts "
+                    "SET status = 'expired', updated_at = datetime('now') "
+                    "WHERE status = 'draft' AND expires_at < datetime('now')"
+                )
+                expired_count = c.rowcount
+                conn.commit()
+            if expired_count:
+                logger.info(f"Cleanup: expired {expired_count} draft(s)")
```

Ruff (0.15.7)

[warning] Logging statement uses f-string

(G004)